### DISCUSSION TITLE
Model: SB-HW20

### DISCUSSION THREAD

**Discussion by wellimaster** [Yogui79/IntexPureSpa#7] [Q&A]
As far as I know (and see), the model **SB-HW20** is ident to the model **28442**. I can't find any number like 28442 on my Pure SPA model, only SB-HW. 

Problem: I can't find the correct channel with the script. No way...so far. 
My idea 1: soldering a wire direct to the serial pins on the SPA board and see whats coming out. Maybe something with the wireless UART connection has changed on the year 2021 model? 
My idea 2: defect LC12s
Question to idea 1: Will it work? Any ideas? 

In another thread you wrote, you used an arduino as a scanner. Can you provide a code?

Cheers,
Martin (Austria)

**Comment by LEECHER1** [comment]
Hey Martin,

Your are from Austria?

The Number you find it on the Bill, Packing or in the shop you have it.

But the SB-HW20 is the 28442 (6seats) or 28440 (4seats) 

Please check the Pins LC12s to ESP (it works correctly?) 
 
Check/Search the Network ID?  And than search channel. 

idea is good but you damage your Controller. I had this. (I have 2 Controllers ;) ) It is very hard to open is good waterproof.

The 2021 is not supported in this Code, only the 2020.

Idea 2: is possible. I have also a damage LC12s but it's my fault. You can buy a new one and check it out. (I living in Austria, Linz) 

Best regards 
Andreas

**Reply by Yogui79** [reply]
Hello 

Please post your Arduino IDE monitoring log after you download the Search channel script
